src/features/ci-cd/hooks/service-installs.ts

```diff
@@ -346,40 +370,20 @@ hooks.addPureHook('PATCH', 'resin', 'device', {
 	},
 });
 
-const addSystemAppServiceInstallHooks = (
-	fieldName: 'should_be_managed_by__release' | 'should_be_operated_by__release',
-) => {
-	hooks.addPureHook('POST', 'resin', 'device', {
```

Comment:
Moved this higher, in the `hooks.addPureHook('POST', 'resin', 'device')` that we have for user apps.